package.json

```diff
@@ -38,12 +38,25 @@
     "format:fix": "prettier --write \"./**/*.{ts,md}\"",
     "prepare": "husky install"
   },
+  "dependencies": {
+    "@tinyhttp/app": "^2.0.25",
+    "@tinyhttp/cors": "^2.0.0",
+    "@tinyhttp/logger": "^1.3.0",
+    "body-parser": "^1.20.1",
+    "cors": "^2.8.5",
+    "sirv": "^2.0.2"
```

Comment:
Use @tinyhttp/cors and milliparsec instead of cors and body-parser

Reply:
let me change these first

Reply:
@talentlessguy replacing body-parser seems to break some stuffs . Maybe we can do it later

Reply:
let me do this . But some function signatures were uncompatible. Had this on cors option as well

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cors/index.d.ts#L23 vs https://github.com/tinyhttp/cors/blob/master/src/index.ts#L5

Reply:
https://github.com/tinyhttp/milliparsec/blob/master/src/index.ts#L44 . The methods on millisec doesnt seem to take options param. But if you look at body-parser all the methods have options parameter. https://github.com/expressjs/body-parser/blob/master/lib/types/json.js

Reply:
you use urlencoded: true which is the same behavior as `json()`

Reply:
Yeah probably missed that part. I just checked the express adapter and implemented like that in current commit. Basically the bodyparser will take configs from nest app instead of hardcoding on the adpater